packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/toolbar/design.md

```diff
@@ -0,0 +1,94 @@
+
+### When not to use
+The toolbar should not be used when the data view already includes the necessary controls to manipulate the data set.
+
+### Layout considerations
+![layout](img/toolbar-layout.png)
+
+The toolbar should appear below the page title, if one exists. It should always appear above the data set to which it applies. Toolbars should stretch to the width of the data view to which it applies, with the elements aligned as specified below. When filters are applied, the toolbar will expand in height to make space for a row of filter chips. Upon clearing the applied filters, the toolbar will collapse to it’s default height.
+
+Within the toolbar, layouts may vary depending on the use case. When used, bulk selection should be the left-most element. Filters and global actions may follow. When pagination is used it should be the right-most element. When pagination is not used, the item count should be the right-most element.
+
+## Variations
+**Basic toolbar**
+![basic](img/basic-toolbar.png)
+The basic toolbar is sufficient for most cases. It includes filters, global actions, and item count. The Toolbar is shown here with an Attribute-Value (textbox) filter, but a variety of filter types may be used.
+1. **Item count**: When a filter is applied, the item count will be updated to represent the filtered subset of items. For example, “10 of 17 items.” The item count should be the right-most element.
+
+**Toolbar with pagination**
+![pagination](img/toolbar-pagination.png)
+When pagination is needed, a pagination component can be used.
+1. **Pagination**: When pagination is used, the pagination component replaces the item count. It should be the right-most element.
+
+**Toolbar with filter chips**
+![filterchips](img/toolbar-filter-chips.png)
+When the current settings of the controls are not sufficient to convey applied filters, filter chips should be used.
+1. **Filter chips**: When filters are applied the toolbar will expand and filter chips will be displayed on the second row. Users can clear all filters by clicking the “Clear filters” link, and the toolbar will collapse back into one row.
+
+**Custom toolbar**
+![custom](img/custom-toolbar.png)
+When the current settings of the controls are not sufficient to convey applied filters, filter chips should be used.
+1. **Filter chips**: When filters are applied the toolbar will expand and filter chips will be displayed on the second row. Users can clear all filters by clicking the “Clear filters” link, and the toolbar will collapse back into one row.
+
+**Mobile toolbar**
+
+Collapsed
+
+![closed](img/mobile-basic.png)
+
+Expanded
+
+![open](img/mobile-expanded.png)
+
+Footer with pagination
+
+![footer](img/mobile-footer.png)
+
+The toolbar adapts to smaller viewport sizes by collapsing or hiding elements that take significant space, including filters and actions. On mobile devices, when used, pagination will be available from the footer, only.
```

Comment:
I would move this so it appears above the mockups. Maybe show this in the context of a mobile screen?

Reply:
There's a mobile screen in the examples section -- should I add another here, or perhaps a schematic?

Reply:
Yeah, or maybe this does not even need to be in both places. The responsive behavior is baked into the component so I think the only value here is to show a designer what to expect if they are on mobile. The example does that pretty well. You could add any text or explanation there.